### ISSUE TITLE
svtplay-dl crashed 

### ISSUE THREAD

**Issue by uwemock** [spaam/svtplay-dl#1120]
### svtplay-dl versions:

svtplay-dl 2.1

### Operating system and Python version:

Kubuntu 18.04.1
Python 3.6.8

### What is the issue:

./svtplay-dl --verbose --proxy ******* https://www.svtplay.se/video/22755582/dieter-rams-designpionjar
DEBUG [1564561693.9738631] ./svtplay-dl/svtplay_dl/utils/getmedia.py/get_media: version: 2.1
DEBUG [1564561693.9753628] ./svtplay-dl/svtplay_dl/service/__init__.py/__init__: service: svtplay
DEBUG [1564561693.9754565] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'https://www.svtplay.se/video/22755582/dieter-rams-designpionjar'
DEBUG [1564561693.977146] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): www.svtplay.se
DEBUG [1564561694.711693] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: https://www.svtplay.se:443 "GET /video/22755582/dieter-rams-designpionjar HTTP/1.1" 200 35754
DEBUG [1564561694.8074784] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://api.svt.se/videoplayer-api/video/1393040-001A'
DEBUG [1564561694.81035] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTP connection (1): 10.7.7.254
DEBUG [1564561695.092296] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://api.svt.se/videoplayer-api/video/1393040-001A HTTP/1.1" 200 744
DEBUG [1564561695.0931988] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-lowbw.m3u8'
DEBUG [1564561695.2271023] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-lowbw.m3u8 HTTP/1.1" 302 0
DEBUG [1564561695.3575397] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2.secure.footprint.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-lowbw.m3u8 HTTP/1.1" 200 1413
DEBUG [1564561695.3585758] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/lowbw.mpd'
DEBUG [1564561695.4734097] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/lowbw.mpd HTTP/1.1" 302 0
DEBUG [1564561695.6148686] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2.secure.footprint.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/lowbw.mpd HTTP/1.1" 200 10289
DEBUG [1564561695.6155555] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4-lowbw.m3u8'
DEBUG [1564561695.728679] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4-lowbw.m3u8 HTTP/1.1" 302 0
DEBUG [1564561695.863032] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2.secure.footprint.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4-lowbw.m3u8 HTTP/1.1" 200 1426
DEBUG [1564561695.863864] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4.m3u8'
DEBUG [1564561695.981706] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4.m3u8 HTTP/1.1" 302 0
DEBUG [1564561696.158846] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2o.akamaized.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/master-fmp4.m3u8 HTTP/1.1" 200 546
DEBUG [1564561696.1594777] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/manifest.mpd'
DEBUG [1564561696.2754667] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://switcher.cdn.svt.se/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/manifest.mpd HTTP/1.1" 302 0
DEBUG [1564561696.4158452] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2.secure.footprint.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/manifest.mpd HTTP/1.1" 200 11709
DEBUG [1564561696.4164026] ./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'http://svt-vod-2o.akamaized.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/manifest.mpd?alt=http%3A%2F%2Fswitcher.cdn.svt.se%2Fb0737d64-4ad8-4d2c-862a-3aac0e9577b5%2Fmanifest.mpd'
DEBUG [1564561696.5825195] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: http://10.7.7.254:8118 "GET http://svt-vod-2o.akamaized.net/d0/se/20190627/b0737d64-4ad8-4d2c-862a-3aac0e9577b5/manifest.mpd?alt=http%3A%2F%2Fswitcher.cdn.svt.se%2Fb0737d64-4ad8-4d2c-862a-3aac0e9577b5%2Fmanifest.mpd HTTP/1.1" 200 11709
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "./svtplay-dl/__main__.py", line 13, in <module>
  File "./svtplay-dl/svtplay_dl/__init__.py", line 64, in main
  File "./svtplay-dl/svtplay_dl/utils/getmedia.py", line 58, in get_media
  File "./svtplay-dl/svtplay_dl/utils/getmedia.py", line 107, in get_one_media
  File "./svtplay-dl/svtplay_dl/service/svtplay.py", line 86, in get
  File "./svtplay-dl/svtplay_dl/service/svtplay.py", line 122, in _get_video
  File "./svtplay-dl/svtplay_dl/fetcher/dash.py", line 143, in dashparse
  File "./svtplay-dl/svtplay_dl/fetcher/dash.py", line 171, in _dashparse
  File "./svtplay-dl/svtplay_dl/fetcher/dash.py", line 102, in adaptionset
IndexError: list index out of range

(Same thing without the proxy.)

**Comment by Sopor** [comment]
Dupe of #1065 #1068 #1070 #1075 #1080 #1084 #1085 #1091 #1093 #1094 #1095 #1102 #1107 #1109 #1112 #1113 #1116

Windows use latest [master](https://ci.appveyor.com/project/spaam/svtplay-dl/builds/23825288/artifacts) and it will work.

If you are running Linux, BSD or macOS use [this](https://svtplay-dl.se/install/source/).

macOS can try follow [this](https://github.com/spaam/svtplay-dl/issues/1070#issuecomment-488087551) guide.

**Comment by uwemock** [comment]
I tried this as well with the same result.

**Comment by Sopor** [comment]
You are still on svtplay-dl 2.1. If you were using latest master it should have been a different version number.

It works perfect for me
```
svtplay-dl.exe https://www.svtplay.se/video/22755582/dieter-rams-designpionjar
INFO: Selected to download hls, bitrate: 3145
INFO: Outfile: dieter.rams.designpionjar.s01e01.dieter.rams.designpionjar-20b64a8-svtplay.audio.ts
[326/326][================================================================================] ETA: 0:00:00
INFO: Outfile: dieter.rams.designpionjar.s01e01.dieter.rams.designpionjar-20b64a8-svtplay.ts
[183/326][============================================....................................] ETA: 0:01:35
```

**Comment by uwemock** [comment]
I had cloned the GitHub repo and built it from there. For some reason, it still preferred the versioin from my ~/bin directory. I deleted that one, and now it looks like it's working.

**Comment by uwemock** [comment]
Sorry... no sound after 10s.

**Comment by Sopor** [comment]
ffmpeg need to be 4 or higher ...

**Comment by spaam** [comment]
the crash is fixed in 2.2. please test it. if its stilll an issue. open a new issue . thanks for the report <3
